app/(main)/layout.tsx

```diff
@@ -14,6 +15,7 @@ export default function MainLayout({
           <Navbar />
           {children}
           <Footer />
+          <LifiWidgetModal />
```

Comment:
Why are we declaring this here? Why not inside Navbar? I think we should declare where it's being used to be easier to find and understand.

Reply:
Like the settings popover? I like this suggestion

Reply:
Yeah, it's better